.github/workflows/nightly.yml

```diff
@@ -85,11 +85,7 @@ jobs:
       fail-fast: false
       matrix:
         python_version: [3.6, 3.7, 3.8]
-        pytorch_version: [1.3, 1.4, 1.5, 1.6, 1.7, 1.8]
-        exclude:
-          # excludes PT 1.3 as it is missing on pypi
-          - python_version: 3.8
```

Comment:
will we still exclude python versions? Here and in other files?

Reply:
we do not exclude py version just combination py3.8 and pt 1.3 as such packages were never published by pytorch